src/gui/nav/DrawerMenu.js

```diff
@@ -43,15 +47,39 @@ export class DrawerMenu implements MComponent<Attrs> {
 					colors: ButtonColors.DrawerNav
 				})
 				: null,
-			logins.isUserLoggedIn() && logins.getUserController().isPremiumAccount()
-				? m(ButtonN, {
+			m(ButtonN, attachDropdown(
+				{
+					label: "supportMenu_label" ,
```

Comment:
I am not sure if this is the appropriate label for the free users, but I can't find anything more suitable.

Reply:
I am definitively sure that the current label isn't appropriate, since that the text is cut and the text shown "Keyboard shortc...". What would you suggest?

Reply:
The last argument to `attachDropdown()` is dropdown width, you can increase it to be something like 400.
We have a `showHelp_action` but translation is slightly off, I think you can use it and we will change translation, just shouldn't forget to do it.